deploy/manifests/browser/base/api.deployment.yaml

```diff
@@ -43,16 +43,16 @@ spec:
                   key: ips
             # Global rate limiting
             - name: MAX_CONCURRENT_ES_REQUESTS
-              value: '100'
+              value: '10'
```

Comment:
Since it looks like this commit just undoes changes made in the immediately previous commit, let's merge this one into the previous one. I made Elissa do it, fair's fair. 😅